### PR TITLE
Fixing and reenabling ServiceBus templates

### PR DESCRIPTION
**Pull request by mathewc**

### CODE REVIEW

File: Bindings/bindings.json [hunk 1]
```diff
@@ -523,6 +565,14 @@
...
                     "help": "This is the connection string for your notification hub namespace."
+                },
+                {
+                    "name": "tagExpression",
+                    "value": "string",
+                    "defaultValue": "tag1||tag2",
+                    "required": false,
+                    "label": "Tag expression (optional)",
+                    "help": "Tag expressions enable you to target specific sets of devices/registrations, when sending a push notification through Notification Hubs. Leave this blank to broadcast to all registered devices<a target='_blank' href='http://go.microsoft.com/fwlink/?LinkId=785094'>More information</a>."
                 }
```
Comment (soninaren) [inline Bindings/bindings.json]: This is an unrelated change

File: Bindings/bindings.json [hunk 2]
```diff
@@ -501,14 +551,6 @@
             "direction": "out",
             "settings": [
                 {
-                    "name": "tagExpression",
```
Comment (mathewc) [inline Bindings/bindings.json]: Looks like a bad merge/rebase. I'll figure it out.

Comment (mathewc) [inline Bindings/bindings.json]: Looks like a bad merge/rebase. I'll figure it out.

File: Templates/ServiceBusQueueTrigger-NodeJS/index.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+﻿module.exports = function(context, myQueueItem) {
+    context.log('Node.js ServiceBus queue trigger function processed message', myQueueItem);
+    context.done(null, OutPutQueueItem);
```
Comment (soninaren) [inline Templates/ServiceBusQueueTrigger-NodeJS/index.js]: There is no Output binding 
Remove OutPutQueueItem

File: Bindings/bindings.json (hunk 1 above)
Comment (mathewc) [inline Bindings/bindings.json]: I accidentally have this PR against master. I'll close and reopen.
